### ISSUE TITLE
The limitation in the training process

### ISSUE THREAD

**Issue by newplay** [QuantumLab-ZY/HamGNN#4]
I would like to inquire whether there are restrictions on the training data for HamGNN, such as requiring it to be of the same composition, like CrI3, CrI4, Cr2I6, but not allowing combinations like CrI3O2（just an example）. This is because DeepH has such a limitation (it cannot train materials with two different compositions).

**Comment by QuantumLab-ZY** [comment]
HamGNN currently has no restrictions on the composition of the training data, which can consist of varying numbers of elements. In our article, we have successfully trained HamGNN using 5K different molecules from the QM dataset. By the way, CrI3, CrI4, and Cr2I6 are all magnetic materials. HamGNN is currently unable to fit the Hamiltonian matrix of magnetic materials. In another work (https://arxiv.org/abs/2306.01558), we have developed a network called HamGNN++, which can handle magnetic systems.
